sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineImpl.java

```diff
@@ -1967,6 +1970,39 @@ public boolean isHibernationEnabled() {
             && ResourceManagerUtils.toPrimitiveBoolean(this.innerModel().additionalCapabilities().hibernationEnabled());
     }
 
+    @Override
+    public SecurityTypes securityType() {
+        SecurityProfile securityProfile = this.innerModel().securityProfile();
+        if (securityProfile == null) {
+            return null;
+        }
+        return securityProfile.securityType();
+    }
+
+    @Override
+    public Boolean isSecureBootEnabled() {
+        if (securityType() == null) {
+            return false;
+        }
+        UefiSettings uefiSettings = this.innerModel().securityProfile().uefiSettings();
+        if (uefiSettings == null) {
+            return false;
+        }
+        return uefiSettings.secureBootEnabled();
+    }
```

Comment:
Could it return `boolean`?

There is multiple `false` return, then last one could be `null`?

Same for below `isVTpmEnabled`.